packages/rum/src/domain/record/observers.ts

```diff
@@ -91,7 +93,7 @@ interface ObserverParam {
   visualViewportResizeCb: VisualViewportResizeCallback
   inputCb: InputCallback
   mediaInteractionCb: MediaInteractionCallback
-  styleSheetRuleCb: StyleSheetRuleCallback
+  cssRulesCb: StyleSheetRuleCallback
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏hum, not great if we end up mixing `css` and `stylesheet` namings 🤔
wdyt?

Reply:
But you just moved the problem:
https://github.com/DataDog/browser-sdk/blob/7c141506176099d07c003142a2f99d73f1976993/packages/rum/src/domain/record/observers.ts#L69

Reply:
`StyleSheetRule` is the type of the record we create. I don't know if we can rename the whole typing of this record 🤔